in-app-wallet/typescript/src/in_app_wallet_examples.ts

```diff
@@ -0,0 +1,157 @@
+import { 
+    Connection, 
+    JsonRpcProvider, 
+    TransactionBlock,
+    SuiTransactionBlockResponse
+} from "@mysten/sui.js";
+import { rpcClient } from "typed-rpc";
+
+// The Key Service is Shinami's secure and stateless way to get access to the In-App Wallet
+const KEY_SERVICE_RPC_URL = "https://api.shinami.com/key/v1/<API_ACCESS_KEY>";
```

Comment:
Since we want to encourage people to use the `X-API-Key` header for these backend services, I wonder if that's easy to do with `typed-rpc`.